style.css

```diff
@@ -0,0 +1,379 @@
+  letter-spacing: 0.03em;
+  line-height: 16px;
+}
+
+button {
+  transition: all 0.5s ease;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #6070ff;
+  text-align: center;
+  padding: 12px 16px;
+  border-radius: 8px;
+  border: none;
+}
+
+button:hover {
+  background: #6070ff;
+  color: #fff !important;
+}
+
+.btn {
+  border: 1px solid #6070ff !important;
+  border-radius: 16px;
+  padding: 12px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 17px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  background: none;
+  color: #6070ff;
+}
+
+/* ---about section---- */
+
+.about-section {
+  border-radius: 0 100px 0 0;
+  background-color: #fff;
+  margin-top: 100px;
+}
+
+.about-section-box {
+  display: flex;
+  flex-direction: column;
+  padding-top: 142px;
+  padding-bottom: 142px;
+}
+
+.resume {
+  width: 50%;
+  margin: 22px 0 48px;
+}
+
+.language-section > h4 {
+  font-weight: 500;
+  line-height: 24px;
+  font-size: 20px;
+  color: #000;
+}
+
+.language-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.language-section:not(:first-child) {
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.drop-down {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  border: none;
+  padding: 23px 8px;
+  font-size: 20px;
+  line-height: 24px;
+  background-color: #fff;
+}
+
+.drop-down > .glyphicon {
+  font-size: 16px;
+  color: #acb7c3;
+}
+
+.language-section-box-sec {
+  display: flex;
+  flex-direction: column;
+  align-self: stretch;
+}
+
+.language-section-box {
+  display: flex;
+  align-items: center;
+  padding: 5px 0 5px 5px;
+  width: 100%;
+  margin-top: 12px;
+  background-color: #f7f7f9;
+  border-radius: 5px;
+}
+
+.language-name {
+  margin-bottom: 0;
+  margin-left: 24px;
+}
+
+.language-section-container {
+  display: flex;
+  flex-direction: column;
+  padding: 0 20px;
+}
+
+.language-title-section {
+  display: flex;
+  justify-items: flex-end;
+  align-self: stretch;
+}
+
+/* --contact-section-- */
+.contact-form {
+  background-color: #6070ff;
+  background-image: url("./images/contact\ form\ shapes-mobile.svg");
+  background-repeat: no-repeat;
+  border-radius: 80px 0 0 0;
+  background-position-x: 100%;
+  background-position-y: 30px;
+}
+
+.contact-form-section {
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-form-card {
+  text-align: center;
+  color: #fff;
+}
+
+.contact-form-title {
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+}
+
+.contact-form-description {
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-left: 15px;
+  margin-right: 10px;
+}
+
+form {
+  margin-top: 40px;
+  display: flex;
+  flex-direction: column;
+}
+
+label {
+  display: inline-block;
+  width: 90px;
+  text-align: right;
+}
```

Comment:
- [ ] [_OPTIONAL_] By setting fixed width it's creating a design difference with the template. Kindly style the section so that it looks similar to the design.

Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/154270295-265b9f9f-2b2b-40e8-bbb9-e42e50b9ecde.png) | ![image](https://user-images.githubusercontent.com/59416136/154270363-51929af4-3ec4-4d5e-9adf-ef86eabce6cd.png)


Kindly fix it.